src/glue/wasm/i64.ts

```diff
@@ -23,7 +23,7 @@ function i64_is<T>(value: T): bool {
 // @ts-ignore: decorator
 @global @inline
 function i64_new(lo: i32, hi: i32 = 0): i64 {
-  return lo | (hi << 32);
+  return <i64><u32>lo | (<i64>hi << 32);
```

Comment:
A spectacular piece of code that was completely wrong before, and looks absolutely bogus now. Apart from the missing cast to `i64` to ensure proper bit size, the `<i64><u32>` is necessary so the compiler doesn't attempt to sign-extend due to the cast value being signed. Somewhat strange, and made me wonder whether we can do better, but without any deeper changes that's the awesome fix fixing things.